### PR TITLE
Chore: Avoid copying package.json into dist folder

### PR DESCRIPTION
**Pull request by andresmgot**
**What is this feature?**

Avoids a warning message about duplicated package.json(s). As far as I can see, that duplicated package.json is not used anywhere.

### CODE REVIEW

File: packages/grafana-plugin-configs/webpack.config.ts [hunk 1]
```diff
@@ -179,18 +183,6 @@ const config = async (env: Record<string, unknown>): Promise<Configuration> => {
...
         },
-        {
-          dir: path.resolve(DIST_DIR),
-          files: ['package.json'],
-          rules: [
-            {
-              search: `"version": "${getPackageJson().version}"`,
-              replace: env.commit
-                ? `"version": "${getPackageJson().version}-${env.commit}"`
-                : `"version": "${getPackageJson().version}"`,
```
Comment (jackw) [inline packages/grafana-plugin-configs/webpack.config.ts:190]: Does this impact the gh workflow that releases an external plugin in someway? 🤔

Comment (andresmgot) [reply packages/grafana-plugin-configs/webpack.config.ts:190]: I checked and it shouldn't: it gets the version from the plugin.json and for the rest, it should be like any other external plugin release.

Comment (jackw) [reply packages/grafana-plugin-configs/webpack.config.ts:190]: Fair enough. Not sure how this made its way here then. 🤷

**Review by jackw — APPROVED** [review]
LGTM! 🚀

**Review by mckn — APPROVED** [review]
LGTM!

**Pull request merged**